src/app/unlocked-stage/unlocked-stage.component.scss

```diff
@@ -1,5 +1,7 @@
 @import "../styles/main";
 
+$gif-height: 500px;
```

Comment:
Is this responsive?
Better to have % units

Reply:
**%** was not working properly. I converted into **em**'s which is also a relative unit and would be responsive.